src/main.ts

```diff
@@ -40,8 +41,8 @@ function bombadilToTomlTestAtomicValue(input: toml.TomlAtomicValue) {
     }
 }
 
```

Comment:
we should either copy your new fixed version of bombadilToTomlTest here, of get rid off this file and the scripts. I dont think toml-test is being mantained, so prefer the later option.